cmd/migration/v0.9.1-v1/v091-domain/market_domain.go

```diff
@@ -0,0 +1,277 @@
+package v091domain
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/shopspring/decimal"
+)
+
+const BalancedReservesType = 1
+
+type Market struct {
+	AccountIndex        int
+	BaseAsset           string
+	QuoteAsset          string
+	BaseAssetPrecision  uint
+	QuoteAssetPrecision uint
+	Fee                 int64
+	FixedFee            FixedFee
+	Tradable            bool
+	Strategy            MakingStrategy
+	Price               Prices
+}
+
+func (m Market) AccountName() string {
+	buf, _ := hex.DecodeString(m.BaseAsset + m.QuoteAsset)
+	return hex.EncodeToString(btcutil.Hash160(buf))[:5]
+}
```

Comment:
this is a "hack", isn't it? this method does not exist in domainv0, correct? totally fine to be here anyway :)

Reply:
I understood this is how we populate label in v1?

Reply:
yeah it's fine.. i was just saying that this method does not really exist in v0.9 domain... nevermind